src/wp-settings.php

```diff
@@ -347,6 +347,7 @@
 require ABSPATH . WPINC . '/block-supports/spacing.php';
 require ABSPATH . WPINC . '/block-supports/typography.php';
 require ABSPATH . WPINC . '/block-supports/settings.php';
+require ABSPATH . WPINC . '/navigation-fallback.php';
```

Comment:
Sorry about this! Thank you for fixing.

Reply:
All's well that ends well. 🍺

Reply:
This is an(other) lesson in why tests are _essential_.

Reply:
You said it!